index.bs

```diff
@@ -357,9 +357,40 @@ is defined by running the following steps:
 
 The <dfn>writeDatagrams</dfn> algorithm is given a |transport| as parameter and
 |data| as input. It is defined by running the following steps:
-1. TODO
-1. Enqueue data for datagram transmission.
-1. Return [=a promise resolved with=] undefined.
+1. Let |timestamp| be a timestamp representing now.
+1. Let |promise| be a new promise.
+1. If |data| is not a {{Uint8Array}} object, then return [=a promise rejected with=] a {{TypeError}}.
+1. Let |bytes| be a copy of bytes which |data| represents.
+1. Let |chunk| be a tuple of |bytes|, |timestamp| and |promise|.
+1. Enqueue |chunk| to |transport|'s {{[[OutgoingDatagramsExpirationDuration]]}}.
```

Comment:
I don't understand this part. Should it be `{{[[OutgoingDatagramsQueue]]}}` instead?

Reply:
Thanks, fixed.